spec/rubocop/cop/performance/redundant_match_spec.rb

```diff
@@ -111,4 +111,15 @@ def method(str)
                    ^^^^^^^^^^^^^^^^^^ Use `=~` in places where the `MatchData` returned by `#match` will not be used.
     RUBY
   end
+
+  it 'autocorrects Regexp#match to Regexp#=~' do
+    expect_offense(<<~TEXT)
+      something if /regex/.match(str)
+                   ^^^^^^^^^^^^^^^^^^ Use `=~` in places where the `MatchData` returned by `#match` will not be used.
+    TEXT
```

Comment:
nits:

```suggestion
    expect_offense(<<~RUBY)
      something if /regex/.match(str)
                   ^^^^^^^^^^^^^^^^^^ Use `=~` in places where the `MatchData` returned by `#match` will not be used.
    RUBY
```

Reply:
I'm wondering if it's not a good idea to describe something that is not Ruby code as being "RUBY".

Modern editors (e.g. Visual Studio Code) recognize this heredoc delimiter as a language name and do syntax highlighting, but this heredoc includes `^^^... `, which is clearly not Ruby code and therefore it fails to parse

![image](https://user-images.githubusercontent.com/111689/181173750-85ce74b5-baf8-41b4-82eb-05522ef977c5.png)

then syntax highlighting for the rest of code will be broken.

Reply:
I understand what you want to say. However, I think it's better to adapt to it because `RUBY` is used by convention.

Reply:
Okay, certainly it is more important thing.
Thanks for having the courage to point this out 👍